src/emc/calcs/wind.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+def uv_to_speed(u, v, direction=False):
+    """
+    Calculates the wind speed from u and v components.
+    Will calculate direction as well if parameter direction = True.
+    
+    Takes into account the wind direction coordinates is different than
+    the trig unit circle coordinate. If the wind direction is 360,
+    then return zero.
+    Parameters
+    ----------
+    u, v: array_like
+        u (easterly) and v (northerly) wind component.
+    direction: boolean
+    Returns
+    -------
+    Wind speed (and direction if direction=True)
+    """
+    if isinstance(u, list) or isinstance(v, list):
```

Comment:
is there any harm in just doing u = np.array(u)? it will do nothing if it is already a numpy array, right?

Reply:
I believe what this does is just make sure that if the data is a list, it makes it a np.array, but I don't see an issue with just doing `u = np.array(u)`.